CefSharp.BrowserSubprocess.Core/JavascriptMethodHandler.h

```diff
@@ -13,19 +13,20 @@ namespace CefSharp
     {
     private:
         gcroot<Func<array<Object^>^, BrowserProcessResponse^>^> _method;
-        gcroot<JavascriptCallbackRegistry^> _callbackRegistry;
+        gcroot<JavascriptCallbackRegistry^> _callbackRegistryReference;
```

Comment:
previously the callbackRegistry has been deleted withi nthe destructor but this is a shared instance among many instances so if one deletes this registry other callbacks will not work anymore.
```
 ~JavascriptMethodHandler()
        {
            delete _method;
            delete _callbackRegistry;
        }
```
https://github.com/cefsharp/CefSharp/blob/master/CefSharp.BrowserSubprocess.Core/JavascriptMethodHandler.h#L28

I have renamed the memeber to make clear that this is only a reference which should not be deleted by this class.